### PR TITLE
Adds '*' support for table-qualified column select

### PR DESCRIPTION
**Pull request by NeedleInAJayStack**
This adds '*' support to table-qualified columns in selects that mirrors the non-table-qualified column selects. This is mainly a convenience.

For example, before this PR `sql.select().column(table: "table", column: "*")` would resolve to ```SELECT `table`.`*` ```, where '*' was an identifier which was invalid in most cases. Now it will resolve to ```SELECT `table`.* ```

Previously to make this query work, it would have to be `sql.select().column(SQLColumn(SQLLiteral.all, table: SQLIdentifier("table"))`, which is kind of cumbersome.

### CODE REVIEW

**Comment by NeedleInAJayStack** [comment]
@0xTim Hey, I don't seem to be able to assign reviewers to pull requests - is the process for contributing to wait for a maintainer to assign themselves to the review?

On a related note, I'd be interested in joining as a maintainer if help is needed.

Thanks!

**Review by 0xTim — APPROVED** [review]
Thanks!

**Comment by 0xTim** [comment]
@NeedleInAJayStack thanks for the bump, this had mostly dropped off my radar! Contact me on Discord but becoming a maintainer would be awesome!

**Comment by NeedleInAJayStack** [comment]
@0xTim Thanks! I'll definitely jump on the discord channels.

**Pull request merged**
